### ISSUE TITLE
Search Box 

### ISSUE THREAD

**Issue by yckmd** [dasevilla/rdiodj#146]
When typing into the search box to find a track, if you leave to another tab - to confirm spelling of a track let's say - and then return and click back into the search box, the contents refresh.

Often I leave to double-check an album title, track title, spelling of an artist, etc. only to have to re-typle it all when the form clears upon re-click. 

Just a heads up from a big fan of y'alls party!

**Comment by FreeForm-Radio** [comment]
The way around this is not to click when you return to the party page. The focus is still on the search box, so don't click, just type. :-)

**Comment by yckmd** [comment]
True - but often the searches that have appears thus far (if you've started to type something in) have obscured what has been entered and then I don't know where I've left off.

So maybe my issue is just more the way that those searched-tracks pop up and obscure the search-bar, then. If that makes any sense.

**Comment by FreeForm-Radio** [comment]
That does make sense and I do the same thing more often than not. 

My problem with the search box is not being able to find the particular track that I want to play. Sometimes entering the lp name works if it is unique enough. Such is not always the case. I'm not sure how the search box could be expanded to show more titles. Perhaps a scroll bar needs to be added with more results displayed.

**Comment by yckmd** [comment]
Yeah - currently it grabs from the "popular songs" list when you type an artist in - I find typing the artist and then a word from the song title or name of the album helps when trying to narrow down, but you're right, it'd be nice to have a little scroller or maybe have the results appear in a different way.

Excuse the MS PAINT - but maybe something like:
![image](https://cloud.githubusercontent.com/assets/12631204/7861688/a5ae98f8-051f-11e5-85a1-848a7dfcec16.png)

**Comment by FreeForm-Radio** [comment]
Thanks for the tip...I'll have to try that.
